<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLogisticaEntregas</class>
 <widget class="QWidget" name="WidgetLogisticaEntregas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <widget class="QLabel" name="labelBuscar">
     <property name="text">
      <string>Buscar:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditBuscar">
     <property name="placeholderText">
      <string>Venda</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="TableView" name="tableCalendario">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="TableView" name="tableCarga">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="2">
          <widget class="QPushButton" name="pushButtonConsultarNFe">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Consultar NF-e</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="pushButtonProtocoloEntrega">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Protocolo Entrega</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="pushButtonConfirmarEntrega">
           <property name="text">
            <string>Confirmar Entrega</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="pushButtonImprimirDanfe">
           <property name="text">
            <string>Imprimir DANFE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="pushButtonCancelarEntrega">
           <property name="text">
            <string>Cancelar entrega</string>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="pushButtonReagendar">
           <property name="text">
            <string>Reagendar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="pushButtonGerarNFe">
           <property name="text">
            <string>Gerar NF-e</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="pushButtonFollowup">
           <property name="text">
            <string>Followup</string>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QPushButton" name="pushButtonObservacao">
           <property name="text">
            <string>Observação</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="TableView" name="tableProdutos">
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tableCalendario</tabstop>
  <tabstop>tableCarga</tabstop>
  <tabstop>pushButtonProtocoloEntrega</tabstop>
  <tabstop>pushButtonGerarNFe</tabstop>
  <tabstop>pushButtonConsultarNFe</tabstop>
  <tabstop>pushButtonConfirmarEntrega</tabstop>
  <tabstop>pushButtonImprimirDanfe</tabstop>
  <tabstop>pushButtonReagendar</tabstop>
  <tabstop>pushButtonCancelarEntrega</tabstop>
  <tabstop>pushButtonFollowup</tabstop>
  <tabstop>tableProdutos</tabstop>
  <tabstop>lineEditBuscar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
